internal/ssh/authorisedkey.go

```diff
@@ -39,3 +40,20 @@ func ParsePublicKey(key string) (PublicKey, error) {
 		Comment: comment,
 	}, nil
 }
+
+// SplitAuthorisedKeys extracts a key slice from the specified key data,
+// by splitting the key data into lines and ignoring comments and blank lines.
+func SplitAuthorisedKeys(keyData string) []string {
```

Comment:
I would argue, that here, it is AuthorizedKeys with a `z`. The Americanz won on thiz one.

Reply:
Specifically, when we refer to SSH Authorized Keys. This is in the SSH package, it is parsing the authorized_keys form of keys that are authorised.

Reply:
Done & Done